corona-warn-app/cwa-app-android
# Family cards (EXPOSUREAPP-12614)

reduce redundancy
fix layout (cause was wrap_content instead of 0dp for the name field)

How to test:
scan different tests for family members
enter a really long name
check the cards

redeemed state is reached when the test is 21 days old and not available on the server, so you can change the environment and time-travel to simulate
